Sources/FoundationEssentials/WinSDK+Extensions.swift

```diff
@@ -157,4 +161,38 @@ package var VOLUME_NAME_DOS: DWORD {
     DWORD(WinSDK.VOLUME_NAME_DOS)
 }
 
+package func PathAllocCombine(_ pszPathIn: String, _ pszMore: String, _ dwFlags: ULONG, _ ppszPathOut: UnsafeMutablePointer<PWSTR?>?) -> HRESULT {
+    pszPathIn.withCString(encodedAs: UTF16.self) { pszPathIn in
+        pszMore.withCString(encodedAs: UTF16.self) { pszMore in
+            WinSDK.PathAllocCombine(pszPathIn, pszMore, dwFlags, ppszPathOut)
+        }
+    }
+}
+
+@_transparent
```

Comment:
Is there a reason to mark these as `@_transparent` rather than letting the compiler make the inlining optimization decisions on its own?

Reply:
These are really supposed to be transparent - they should be imported through the clang importer, but it doesn't understand complex macros.

Reply:
I spoke with @lorentey about this and after touching base with him I think we should use `@inline(__always)` instead of `@_transparent` here. I agree that we shouldn't have to incur a function call for these here, but `@_transparent` also inherently adds `@inlinable` to these declarations which means their implementations could accidentally leak into our interface if we accidentally included them in other inlinable code. `@inline(__always)` should get us the same performance wins here given that these are `internal` declarations without carrying the risk of accidentally letting this implementation detail leak into the interface without an intentional decision around that

Reply:
Okay, seems reasonable enough.